frontend/templates/product/sections/ProductReviewsSection/index.tsx

```diff
@@ -217,15 +219,41 @@ function ReviewsStats({
 
 type ProductReviewLineItemProps = {
    review: ProductReview;
+   variantSku: string | undefined | null;
 };
-function ProductReviewLineItem({ review }: ProductReviewLineItemProps) {
+function ProductReviewLineItem({
+   review,
+   variantSku,
+}: ProductReviewLineItemProps) {
+   const isAdminUser = useAuthenticatedUser().data?.isAdmin ?? false;
+   const appContext = useAppContext();
+
    return (
       <Box
          py="6"
          borderBottomWidth="1px"
          borderColor="gray.200"
          data-testid="product-review-line-item"
       >
+         {isAdminUser && (
+            <Button
+               variant="link"
+               as={Link}
+               bgColor="transparent"
+               textColor="brand"
+               fontFamily="heading"
+               lineHeight="1.29"
+               fontWeight="semibold"
+               fontSize="14px"
+               color="brand.500"
+               textAlign="center"
```

Comment:
Minor: isn't text center aligned in buttons by default?

Reply:
Yes! I was also thinking that, but since it was already present here I left it in
 https://github.com/iFixit/react-commerce/blob/4f9dcdda5c19800960ae9d96827a5fb83fa27dca/frontend/templates/troubleshooting/index.tsx#L204